rpc/client.go

```diff
@@ -80,14 +79,14 @@ func NewHttpRpcClient(rpcServerUrl string) (*RpcClient, error) {
 	return c, nil
 }
 
-func (rc *RpcClient) GetVirtualChannel(id types.Destination) query.PaymentChannelInfo {
-	req := serde.GetPaymentChannelRequest{Id: id}
+func (rc *RpcClient) GetPaymentChannel(chId types.Destination) query.PaymentChannelInfo {
+	req := serde.GetPaymentChannelRequest{Id: chId}
 
 	return waitForRequest[serde.GetPaymentChannelRequest, query.PaymentChannelInfo](rc, serde.GetPaymentChannelRequestMethod, req)
```

Comment:
I'm not 100% sure but it looks like `waitForRequest` will panic in many error conditions, such that we will still crash the go-nitro server if we query for a channel with.a zero id. So still some more work to do to fix that! https://github.com/statechannels/go-nitro/issues/1275.

Reply:
I added [this line](https://github.com/statechannels/go-nitro/pull/1368/files#diff-e411c027b510060c0ba3026138d11ece9c213a5e5927164270fb1f9ed45a8249R180) to test that the server does not panic when passed a non-existent chId. I believe its working as expected.

Reply:
Awesome, I guess that settles it then 👍